src/Utilities/CssConcatenator/MediaRule.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier\Utilities\CssConcatenator;
+
+/**
+ * Implementation of media rules for \Pelago\Emogrifier\Utilities\CssConcatenator
+ *
+ * @internal
+ *
+ * @author Jake Hotson <jake.github@qzdesign.co.uk>
+ * @author SignpostMarv
```

Comment:
felt this was appropriate due to the transposition of `stdClass` casting & `compact()` to dedicated classes, if that's not an issue?

Reply:
By all means, please leave your footprint.  You will be remembered in history :)

If you want to add an email address, you can.  I understand the reasons for not doing so, of course :)

Reply:
email address is in the git log :P

Reply:
> email address is in the git log :P

Not many people will look at that (I never have).  It's entirely up to you whether you want to include it in the source code, and not a decision I'd take too lightly.  Have a think.  Can be added (or removed) later if you change your mind, though if added first then wanted to be removed, could be impossible removing all traces in copies of the repo etc.  So maybe leave it off for now (i.e. no further changes to make for this conversation in this PR), and can be added in later if you like.

Reply:
I'd prefer to have a real name there instead of a nickname, though.